tests/interpolation/built_in_resolvers/test_dict.py

```diff
@@ -234,33 +234,18 @@ def test_dict_values_dictconfig_resolver_output(
     assert cfg.foo[key] == expected
 
 
-@mark.parametrize(
-    ("make_resolver", "expected_value", "expected_content"),
-    [
-        param(
-            lambda _parent_: OmegaConf.create({"a": 0, "b": 1}, parent=_parent_),
-            [0, 1],
-            ["${y.a}", "${y.b}"],
-            id="dictconfig_with_parent",
-        ),
-        param(
-            lambda: {"a": 0, "b": 1},
-            [0, 1],
-            ["${y.a}", "${y.b}"],
-            id="plain_dict",
-        ),
-    ],
-)
-def test_dict_values_transient_interpolation(
+@mark.parametrize("dict_func", ["oc.dict.values", "oc.dict.keys"])
+def test_extract_from_dict_resolver_output(
```

Comment:
```suggestion
def test_extract_from_plain_dict_unsupported(
```
Also, I think we have decided that the only supported input the string keys.
dunno if it makes sense to test something so intricate now that the behavior is so simple. I vote to just remove this test as well.

Reply:
I'm not particularly attached to this test => removed in 9050018
(note: it was still using a string key as input, it's just that the corresponding key was a plain dict -- I think using a resolver is the only way to get a plain dict in a config)